src/Symfony/Component/Console/Command/InvokableCommand.php

```diff
@@ -30,26 +30,28 @@
  */
 class InvokableCommand implements SignalableCommandInterface
 {
-    private readonly \Closure $code;
+    private readonly \Closure $closure;
     private readonly ?SignalableCommandInterface $signalableCommand;
     private readonly \ReflectionFunction $reflection;
     private bool $triggerDeprecations = false;
+    private $code;
 
     public function __construct(
         private readonly Command $command,
         callable $code,
     ) {
-        $this->code = $this->getClosure($code);
+        $this->code = $code;
```

Comment:
```suggestion
```

Reply:
Alas I dont think this will work. See my comment above about `callable` in typed property